app/components/add-layer/template.hbs

```diff
@@ -21,6 +20,7 @@
             {{#bs-popover placement="top"}}Drag and drop a block and create your own content{{/bs-popover}}
         </button>
     </div> 
+    <button draggable="true" ondragend={{action 'dragStop'}} ondragstart={{action 'drag' this}} data-title='navigation' class="fa fa-bars layer-link" href="#" title="Pull navigation"></button>
```

Comment:
Should probably make this a component called "draggable-button" or something, then have a manifest of blockname/icon pairs in the controller to iterate over.

Reply:
ill make a ticket for that